### ISSUE TITLE
Can duplicate barcode input and print be supported?

### ISSUE THREAD

**Issue by dongchch2016** [ExLibrisGroup/spineomatic-cloudapp#71]
It is not supported now, because 'set<string>' is used to store barcodes, but is it possible to add checkbox option to allow repeated barcodes input, so that repeated barcodes can be recorded and printed.

Thanks,
chenchen

**Comment by dongchch2016** [comment]
Hi,Mgobat,

Will this requirement be developed? If so, will there be a specific timeline? Because this is what the customer needs urgently.

Thanks,
chenchen

**Comment by mgobat** [comment]
@dongchch2016, we haven't evaluated the feasibility of this enhancement yet. Therefore, there is no timeline for implementation. Sorry.

**Comment by dongchch2016** [comment]
@mgobat No worries, thank you for your reply.
So in other words, is the requirement acceptable for development?

Thanks,
chenchen

**Comment by dongchch2016** [comment]
@mgobat No worries, thank you for your reply.
So in other words, is the requirement acceptable for development?

Thanks,
chenchen

**Comment by mgobat** [comment]
@dongchch2016, the original developer has left the company. I need to familiarize myself with the application (and the Alma APIs used in it) before I can determine if it can be done.

**Comment by dongchch2016** [comment]
@mgobat OK. No problem.

Thank you!
chenchen

**Comment by dongchch2016** [comment]
Hi,Mgobat,

How is this going? 

Thanks,
chenchen

**Comment by mgobat** [comment]
@dongchch2016, what is the workflow/purpose for this? Item barcodes are unique, so why would you want a barcode in the set multiple times? Thanks.

**Comment by cindywanghui01** [comment]
@mgobat Customer need to print out the same barcodes multiple times because they need to attach both the item itself and the item slipcases. Hope this helps.

Thanks,
Cindy

**Comment by mgobat** [comment]
@cindywanghui01, would an option to print multiple copies of the same set of labels solve the problem instead of multiple occurrences of the same barcode in the set?

**Comment by cindywanghui01** [comment]
@mgobat, it would be better to have an option to have duplicate barcode because this kind of book will mixed with the normal books together, and they have a lot of physical books to deal with everyday, it'll be more efficient if they can just scan the item without doing anything else. Otherwise, they need to pick up those books with slipcases and created a set, it won't be efficient and will cause more work.

BR,
Cindy

**Comment by mgobat** [comment]
@cindywanghui01, thank you for the explanation. We certainly don't want to make their workflow more complicated. Will investigate what we can do. No commitment yet.

**Comment by cindywanghui01** [comment]
@mgobat hi, is there any luck?

BR,
Cindy

**Comment by mgobat** [comment]
@cindywanghui01, I don't think it is an easy/quick change. I have to find something else to use besides Sets.

**Comment by cindywanghui01** [comment]
Hi mgobat,

I am helping customer to check if there is any further update, thanks!

BR,
Cindy

**Comment by mgobat** [comment]
@cindywanghui01, no update yet. Sorry.

**Comment by BarbaraSalfate** [comment]
Hi,
Here in Universidad Católica de Chile need this enhancement too because in the PC spineomatic has the option to scan the same barcode several times.
We will be watching for updates.
Thanks in advance.
Bárbara

**Comment by mgobat** [comment]
If printing multiple copies of the same label set helps, this can be done if you use Microsoft Edge as your browser. When the 'Print" dialog appears after clicking "Print X Labels" in the Cloud App, you can change the number of copies. This will give you the same labels printed for however many times you set the "Copies". Unfortunately, this does not work in Firefox or Chrome.

We still don't have a timeline for scanning the same barcode more than once.

**Comment by kronneCSU** [comment]
In trying to configure this for batch printing, sets seem to make the most sense since both scanning and selecting only let you do one item (or can't select items from multiple records). However, the set doesn't recognize duplicate barcodes when you need to print two of the same label. Any suggestions for how to accomplish this. For the select method it would be nice to be able to select, save and add from another record so you can build up a batch from multiple titles.

For some context, when we process serial items (magazines) we usually process 16 items from different titles at once (this is due to the label sheets we use). The way serial items arrive, we couldn't wait to process 16 of the same title as that would create a huge back up for us and items wouldn't go out to the shelf in a timely manner. When a title has a supplement, we don't generally make a separate item for the supplement and create two labels from the same barcode.

Let me know if there is any other information I can provide.
